dataquality/integrations/seq2seq/core.py

```diff
@@ -171,8 +172,8 @@ def watch(
 
     # A model of the correct type is required if we need to generate
     if generation_splits:
-        assert isinstance(
-            model, PreTrainedModel
+        assert isinstance(model, PreTrainedModel) or isinstance(
+            model, PeftModel
```

Comment:
```suggestion
        assert isinstance(model, (PreTrainedModel, PeftModel))
```

https://stackoverflow.com/questions/33311258/python-check-if-variable-isinstance-of-any-type-in-list

Reply:
Nice!